### DISCUSSION TITLE
Can't connect to mitm.it on Android phone

### DISCUSSION THREAD

**Discussion by SapiensAnatis** [mitmproxy/mitmproxy#5622] [Q&A] [answered]
I am trying to help a friend set up mitmproxy for inspecting Android traffic.

They have `mitmweb` running on their Windows PC on port 8080. They have set the proxy in their Android wi-fi settings to 192.168.0.180 (their PC's local address) with a port of 8080. However, when trying to connect to `mitm.it`, the page doesn't load (time-out?).

On their desktop PC which is running `mitmweb`, they are able to set the proxy to 192.168.0.180:8080 or 127.0.0.1:8080 and connect to mitm.it and download the certificates without issue. So it seems like there's some issue with the PC receiving requests from the phone. They have tried with the default stock Android browser and Chrome.

I'm pretty much out of ideas for troubleshooting steps to suggest, although I do wonder if there might be something going on with Windows Firewall. I'd appreciate it if anyone else had any ideas for things to try.

**Comment by SapiensAnatis** [comment, marked as answer]
This was due to an issue with Windows Firewall -- outbound rule was set to not allow public networks through

**Reply by rfpg1** [reply]
How did you change that outbound rule?

**Comment by gbiz123** [comment]
I was facing a similar issue on Linux where traffic was not being routed properly from my phone to the mitmproxy server running on my computer. Turned out to be the same issue, I had to set a firewall rule allowing port 8080 with `sudo ufw allow 8080`

**Comment by tojestzart** [comment]
mitmweb --set web_host=0.0.0.0 --set web_port=8081
